uaclient/api/api.py

```diff
@@ -28,6 +28,7 @@
     "u.pro.security.status.reboot_required.v1",
     "u.pro.version.v1",
     "u.security.package_manifest.v1",
+    "u.pro.unattended_upgrades.status.v1",
```

Comment:
That's a good question, maybe not, because this API is not for package related information per se.
But I can understand the rationale for the change.

What do you think @renanrodrigo ?

Reply:
Agreed not directly related to packages, voting for keep-as-is

Reply:
That makes sense - resolving